ats/jobs/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from model_utils import Choices
+from django.utils.translation import ugettext_lazy as _
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=25)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = "Category"
+        verbose_name_plural = "Categories"
+
+
+class Job(models.Model):
+    STATUS = Choices(
+        (1, 'DRAFT', _('Draft')),
+        (2, 'ACTIVE', _('Active')),
+        (3, 'SHORT_LISTED', _('Short Listed')),
+        (4, 'REJECTED', _('Rejected')),
+        (5, 'ARCHIVED', _('Archived'))
+    )
+
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    tags = models.TextField()
```

Comment:
This is not wrong, but not the best design, if you want in the future to filter jobs based on the tag, you can't do this because it's a TextField.
Add a Tag model and link the job with the Tag.